spider/scrapy-tsest.py

```diff
@@ -0,0 +1,20 @@
+from apscheduler.schedulers.blocking import BlockingScheduler
+from datetime import datetime
+import os
+import time
+
+def run_spider():
+    print(datetime.now().strftime("%H:%M:%S")," Run the scrapy")
+    os.system("scrapy crawl quotes")
```

Comment:
Can this thing be only run under command line?
I think it's better that the scraper returns the tempurature data via a function and the framework can handle the common part like writing the data to the database.

Otherwise, if the scraped results have to be written into local files, you should define the common format for those files and implement a parser in the main framework.